src/lib/operators/jit_operator/operators/jit_write_tuples.cpp

```diff
@@ -32,7 +32,7 @@ std::shared_ptr<Table> JitWriteTuples::create_output_table(const ChunkOffset inp
 void JitWriteTuples::before_query(Table& out_table, JitRuntimeContext& context) const { _create_output_chunk(context); }
 
 void JitWriteTuples::after_chunk(Table& out_table, JitRuntimeContext& context) const {
-  if (context.out_chunk.size() > 0 && context.out_chunk[0]->size() > 0) {
+  if (context.out_chunk.empty() && context.out_chunk[0]->size() > 0) {
```

Comment:
`!context.out_chunk.empty()` ?!

Reply:
Oups. Good thing that the tests caught it too.